modules/apps/object/object-service/src/main/java/com/liferay/object/internal/scope/CompanyInstanceObjectScopeProviderImpl.java

```diff
@@ -50,6 +50,11 @@ public String[] getRootPanelCategoryKeys() {
 		};
 	}
 
+	@Override
+	public Long getValidGroupId(long companyId, String scopeKey) {
+		return 0L;
```

Comment:
@devpedrotavares can we return `null` here?

Reply:
Yes, I changed it